DaandB/GDEV_Huiswerk1.cs

```diff
@@ -5,7 +5,11 @@
 
 public class playerController : MonoBehaviour {
 
-    public float speed { get; set; } = 10f;
+    [SerializeField] private float _speed { get; set; } = 10f;
```

Comment:
Iets te vroeg met mij approve. Zit nog een klein foutje in.

Op regel 8 heb je
`[SerializeField] private float _speed { get; set; } = 10f;`

Dit moet je veranderen in
`[SerializeField] private float _speed = 10f;`

Reply:
Ayy ik zie het. Zal 't gelijk even veranderen.